pkg/cmd/scan.go

```diff
@@ -230,6 +231,8 @@ func scanRun(opts *pkg.ScanOptions) error {
 		return err
 	}
 
+	globaloutput.Printf(color.WhiteString("Done in %s\n", analysis.Duration.Round(time.Second)))
```

Comment:
Done in %s **seconds**

Reply:
`Duration.Round` already format the duration

![image](https://user-images.githubusercontent.com/16480203/122953154-caff4880-d37e-11eb-94bb-6659b7805dd2.png)

Reply:
Oh cool 👍🏻